crates/wasmtime/src/runtime/vm/libcalls.rs

```diff
@@ -1137,6 +1137,40 @@ unsafe fn check_malloc(
     Ok(0)
 }
 
+// Hook for validating calloc using wmemcheck_state.
+#[cfg(feature = "wmemcheck")]
+unsafe fn check_calloc(store: &mut dyn VMStore, instance: &mut Instance, addr: u32, count: u32, size: u32) -> Result<u32> {
+    check_malloc(store, instance, addr, count * size)
```

Comment:
Can we add a note here that since we aren't tracking undefined-ness, we don't have to care about the other semantic difference between `calloc` and `malloc`, namely that memory is zeroed?